app/jobs/resolution_proofing_job.rb

```diff
@@ -84,9 +84,17 @@ def logger_info_hash(hash)
     logger.info(hash.to_json)
   end
 
-  def add_threatmetrix_result_to_callback_result(callback_log_data_result, threatmetrix_result)
-    callback_log_data_result[:threatmetrix_success] = threatmetrix_result.success?
-    callback_log_data_result[:threatmetrix_request_id] = threatmetrix_result.transaction_id
```

Comment:
I removed these keys, the same information is in this `context.stages.threatmetrix` block and the new block matches the structure of InstantVerify, AAMVA better

Reply:
nice!